app/src/main/java/edu/kmaooad/capstone23/orgs/dal/OrgsRepository.java

```diff
@@ -16,7 +16,11 @@ public Org findByName(String name) {
     }
 
     public Org findById(String id) {
-        return find("id", id).firstResult();
+       try {
+              return findById(new ObjectId(id));
+         } catch (IllegalArgumentException e) {
+              return null;
```

Comment:
you changed interface of a method, though the return type remains the same. But, probably, in other places where it is called, it behaves differently